src/cic/repl.clj

```diff
@@ -194,8 +194,8 @@
   [out-file train-years n-runs seed]
   (let [{:keys [periods placement-costs duration-model placements-model] :as model-inputs} (prepare-model-inputs (load-model-inputs))
         project-from (time/max-date (map :beginning periods))
-        project-to (time/years-after project-from 3)
-        learn-from (time/years-before project-from train-years)
+        project-to (time/years-after project-from 10)
```

Comment:
this magic number seems to change and could use a FIXME with a note saying why it changes